ntbs-service/Middleware/AuditGetRequestMiddleware.cs

```diff
@@ -28,7 +28,7 @@ public async Task InvokeAsync(HttpContext context, IAuditService auditService)
             var request = context.Request;
             var response = context.Response;
 
-            if (response.StatusCode == StatusCodes.Status200OK && request.Method == "GET" && request.Query.ContainsKey("id"))
+            if (response.StatusCode == StatusCodes.Status200OK && request.Method == "GET" && request.Path.ToString().Contains("Notification"))
```

Comment:
Noticed that this was broken since changing the routing!

Reply:
woops! good catch!